### PR TITLE
Done keyboard view added

### PR DESCRIPTION
**Pull request by mahammad-webelight**
Done keyboard view added with custom ( You have the option to hide the done keyboard, and you can enable the previous and next buttons to focus on the next and previous fields. )

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,38 @@ A package that enables you to use a comprehensive set of utilities in a single p
 
 ### This package contains the below points as of now. We can improve and add new points as per requirements in the future
 
+## Done Keyboard View For Android and IOS
+
+    You need to add the below line into FocusNode, which you can manage with Custom Text Filed. You have the option to hide the done keyboard, and you can enable the previous and next buttons to focus on the next and previous fields.
+
+----
+
+    focusNode: showDoneKeyboard
```
Comment (zeeshan-webelight) [inline README.md:36]: is this needed here?

File: example/lib/views/app_text_field.dart [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+              ..addListener(() {
+                final hasFocus = focusNode.hasFocus;
+                if (hasFocus) {
+                  KeyboardOverlay.showOverlay(
+                    context: context,
+                    isNextButton: isNextButton,
+                    isPrevious: isPrevious,
+                    isShowButton: (isNextButton || isPrevious),
+                  );
+                } else {
```
Comment (zeeshan-webelight) [inline example/lib/views/app_text_field.dart:35]: else will be called anytime as we assigned value to hasFocus?

File: lib/src/keyboard_helper/input_done_view.dart [hunk 3]
```diff
@@ -0,0 +1,61 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class InputDoneView extends StatelessWidget {
+  final bool isNext;
+  final bool isPrevious;
+  final bool isShowButton;
```
Comment (zeeshan-webelight) [inline lib/src/keyboard_helper/input_done_view.dart:7]: can we specify `isShowButton` to `isShowDoneButton` ?

File: example/lib/views/app_text_field.dart (hunk 2 above)
Comment (mahammad-webelight) [reply example/lib/views/app_text_field.dart:35]: If it is focused, then it will view the Done View keyboard, and otherwise it will not view

**Pull request merged**